tools/pd-ctl/tests/tso/tso_test.go

```diff
@@ -36,7 +36,7 @@ func TestTSO(t *testing.T) {
 
 	// tso command
 	ts := "395181938313123110"
-	args := []string{"-u", "127.0.0.1", "tso", ts}
+	args := []string{"tso", ts}
```

Comment:
We could keep this to test the case that even if the address given is invalid the command still works.

Reply:
get it